requirements/app/cloud.txt

```diff
@@ -1,3 +1,3 @@
-redis>=4.0.0, <=4.2.4
+redis>=4.0.0, <5.0.0
```

Comment:
Is this meant to be strict?
still, this won't prevent accidental failures in our CI for e.g. 4.3.0